tests/kuttl/common/scripts/check_cluster_status.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# List of Northbound pods
```

Comment:
I think we should repeat the scenario for SB database too. This script should be common to both cases. Just pass the database type (sb / nb) as an argument.

Reply:
Great point! I added this :+1: